<ui version="4.0" >
 <class>NMGNetperfWidgetBase</class>
 <widget class="QWidget" name="NMGNetperfWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Throughtput</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QGroupBox" name="groupBox2" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>400</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>400</width>
             <height>160</height>
            </size>
           </property>
           <property name="title" >
            <string>Test Properties</string>
           </property>
           <widget class="QLabel" name="textLabel2_5" >
            <property name="geometry" >
             <rect>
              <x>11</x>
              <y>113</y>
              <width>87</width>
              <height>24</height>
             </rect>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Source Port:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QLabel" name="textLabel4" >
            <property name="geometry" >
             <rect>
              <x>11</x>
              <y>53</y>
              <width>99</width>
              <height>24</height>
             </rect>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Test Identifier:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QDateEdit" name="dateStartTime" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="geometry" >
             <rect>
              <x>283</x>
              <y>83</y>
              <width>107</width>
              <height>24</height>
             </rect>
            </property>
           </widget>
           <widget class="QTimeEdit" name="timeStartTime" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="geometry" >
             <rect>
              <x>116</x>
              <y>83</y>
              <width>89</width>
              <height>24</height>
             </rect>
            </property>
           </widget>
           <widget class="QComboBox" name="comboGMTOrLocalTime" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="geometry" >
             <rect>
              <x>211</x>
              <y>83</y>
              <width>66</width>
              <height>24</height>
             </rect>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text" >
              <string>Local</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>GMT</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="textLabel1_3" >
            <property name="geometry" >
             <rect>
              <x>11</x>
              <y>23</y>
              <width>81</width>
              <height>24</height>
             </rect>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Test Name:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="chkStartTime" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="geometry" >
             <rect>
              <x>11</x>
              <y>84</y>
              <width>95</width>
              <height>22</height>
             </rect>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&amp;Start time:</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="txtSourcePort" >
            <property name="geometry" >
             <rect>
              <x>116</x>
              <y>113</y>
              <width>89</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimum" >
             <number>5000</number>
            </property>
            <property name="maximum" >
             <number>65000</number>
            </property>
            <property name="value" >
             <number>12865</number>
            </property>
           </widget>
           <widget class="QLineEdit" name="txtTestId" >
            <property name="geometry" >
             <rect>
              <x>116</x>
              <y>53</y>
              <width>274</width>
              <height>24</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="txtTestName" >
            <property name="geometry" >
             <rect>
              <x>116</x>
              <y>23</y>
              <width>274</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text" >
             <string>Throughtput Test</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupTestHosts" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>160</height>
            </size>
           </property>
           <property name="title" >
            <string>Test Hosts</string>
           </property>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="textLabel2" >
                <property name="toolTip" >
                 <string>Management Interface</string>
                </property>
                <property name="text" >
                 <string>Source:</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="NMGHostInformationComboWidget" native="1" name="hostSourceCombo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="textLabel2_2" >
                <property name="toolTip" >
                 <string>Management Interface</string>
                </property>
                <property name="text" >
                 <string>Destination:</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="NMGHostInformationComboWidget" native="1" name="hostDestinationCombo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>0</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>31</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QGroupBox" name="groupBox8" >
           <property name="minimumSize" >
            <size>
             <width>400</width>
             <height>223</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>400</width>
             <height>32767</height>
            </size>
           </property>
           <property name="title" >
            <string>Test Parameters</string>
           </property>
           <widget class="QTabWidget" name="tabWidget3" >
            <property name="geometry" >
             <rect>
              <x>10</x>
              <y>20</y>
              <width>380</width>
              <height>190</height>
             </rect>
            </property>
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <widget class="QWidget" name="General_Parameters" >
             <attribute name="title" >
              <string>&amp;General</string>
             </attribute>
             <widget class="QGroupBox" name="buttonGroup_2" >
              <property name="geometry" >
               <rect>
                <x>10</x>
                <y>10</y>
                <width>250</width>
                <height>110</height>
               </rect>
              </property>
              <property name="title" >
               <string>Duration</string>
              </property>
              <widget class="QSpinBox" name="txtTestDuration" >
               <property name="geometry" >
                <rect>
                 <x>20</x>
                 <y>30</y>
                 <width>110</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="minimum" >
                <number>1</number>
               </property>
               <property name="maximum" >
                <number>999999999</number>
               </property>
               <property name="value" >
                <number>10</number>
               </property>
              </widget>
              <widget class="QRadioButton" name="radSeconds" >
               <property name="geometry" >
                <rect>
                 <x>150</x>
                 <y>30</y>
                 <width>80</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="text" >
                <string>se&amp;conds</string>
               </property>
               <property name="shortcut" >
                <string>Alt+C</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QRadioButton" name="radBytes" >
               <property name="geometry" >
                <rect>
                 <x>150</x>
                 <y>60</y>
                 <width>80</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="text" >
                <string>&amp;bytes</string>
               </property>
               <property name="shortcut" >
                <string>Alt+B</string>
               </property>
              </widget>
             </widget>
             <widget class="QGroupBox" name="buttonGroup2" >
              <property name="geometry" >
               <rect>
                <x>270</x>
                <y>10</y>
                <width>80</width>
                <height>110</height>
               </rect>
              </property>
              <property name="title" >
               <string>Type</string>
              </property>
              <widget class="QRadioButton" name="radTcp" >
               <property name="geometry" >
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>60</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text" >
                <string>TC&amp;P</string>
               </property>
               <property name="shortcut" >
                <string>Alt+P</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QRadioButton" name="radUdp" >
               <property name="geometry" >
                <rect>
                 <x>10</x>
                 <y>60</y>
                 <width>50</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text" >
                <string>&amp;UDP</string>
               </property>
               <property name="shortcut" >
                <string>Alt+U</string>
               </property>
              </widget>
             </widget>
             <widget class="QCheckBox" name="chkNoDelay" >
              <property name="geometry" >
               <rect>
                <x>20</x>
                <y>130</y>
                <width>90</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text" >
               <string>NODELA&amp;Y</string>
              </property>
              <property name="shortcut" >
               <string>Alt+Y</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="Sender_Parameters" >
             <attribute name="title" >
              <string>S&amp;ender</string>
             </attribute>
             <widget class="QWidget" name="layout23" >
              <property name="geometry" >
               <rect>
                <x>17</x>
                <y>25</y>
                <width>300</width>
                <height>110</height>
               </rect>
              </property>
              <layout class="QGridLayout" >
               <item row="1" column="0" >
                <widget class="QCheckBox" name="chkSss" >
                 <property name="text" >
                  <string>Send Soc&amp;ket Size</string>
                 </property>
                 <property name="shortcut" >
                  <string>Alt+K</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" >
                <widget class="QCheckBox" name="chkSms" >
                 <property name="text" >
                  <string>Send Message Si&amp;ze</string>
                 </property>
                 <property name="shortcut" >
                  <string>Alt+Z</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" >
                <widget class="QLineEdit" name="txtSss" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" >
                <widget class="QLineEdit" name="txtSms" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="Receiver_Parameters" >
             <attribute name="title" >
              <string>Recei&amp;ver</string>
             </attribute>
             <widget class="QWidget" name="layout26" >
              <property name="geometry" >
               <rect>
                <x>13</x>
                <y>26</y>
                <width>310</width>
                <height>110</height>
               </rect>
              </property>
              <layout class="QGridLayout" >
               <item row="0" column="1" >
                <widget class="QLineEdit" name="txtRms" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" >
                <widget class="QLineEdit" name="txtRss" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" >
                <widget class="QCheckBox" name="chkRms" >
                 <property name="text" >
                  <string>Receive Message Si&amp;ze</string>
                 </property>
                 <property name="shortcut" >
                  <string>Alt+Z</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" >
                <widget class="QCheckBox" name="chkRss" >
                 <property name="text" >
                  <string>Receive Soc&amp;ket Size</string>
                 </property>
                 <property name="shortcut" >
                  <string>Alt+K</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>340</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>21</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>610</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdStop" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>St&amp;op</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdStart" >
       <property name="text" >
        <string>St&amp;art</string>
       </property>
       <property name="shortcut" >
        <string>Alt+A</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>NMGHostInformationComboWidget</class>
   <extends>QWidget</extends>
   <header>nmghostinformationcombowidget.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >nmghostinformationcombowidget.h</include>
  <include location="local" >nmghostinformationcombowidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>chkSms</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtSms</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSss</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtSss</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRms</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtRms</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRss</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtRss</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkStartTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeStartTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkStartTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboGMTOrLocalTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkStartTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateStartTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radUdp</sender>
   <signal>toggled(bool)</signal>
   <receiver>radBytes</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>328</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel" >
     <x>223</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radUdp</sender>
   <signal>toggled(bool)</signal>
   <receiver>radSeconds</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>328</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel" >
     <x>223</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
